### DISCUSSION TITLE
Does CVE-2021-30551 affect Cefsharp

### DISCUSSION THREAD

**Discussion by d2lip** [cefsharp/CefSharp#3619] [Q&A]
Hi,

Just wondering if the community is looking into the vulnerability that chrome fixed recently as it seems it affects Chromium.
Is this being investigated to see if Cefsharp also suffers from the vulnerability?

**Comment by amaitland** [comment]
> Is this being investigated to see if Cefsharp also suffers from the vulnerability?

Chrome for obvious reasons doesn't provide details to the public and unfortunately I don't have access to the restricted parts of the chromium issue tracker. 

As the issue appears to be in the V8 engine it's safest to assume that CefSharp is vulnerable. 

https://chromereleases.googleblog.com/2021/06/stable-channel-update-for-desktop.html

The fix is included in Chromium 91.0.4472.101,  I upgraded yesterday in commit https://github.com/cefsharp/CefSharp/commit/bd847f844f906813a2e98ef254ccf319e55d9753

All going well I'll release a new version tomorrow. Subscribe to release notifications using the watch button here on GitHub. 

If you need a build before that you can build the https://github.com/cefsharp/CefSharp/tree/cefsharp/91 branch yourself.
